docs/core/domains.rst

```diff
@@ -316,3 +316,29 @@ featurized as normal.
 
     If you really want these entities not to influence action prediction we
     suggest you make the slots with the same name of type ``unfeaturized``.
+
+.. _session_config:
+
+Session configuration
+---------------------
+
+A conversation session represents the dialogue between the assistant and the user.
+Conversation sessions can begin in three ways:
+
+  1. the user begins the conversation with the assistant,
+  2. the user sends their first message after a configurable period of inactivity, or
+  3. a manual session start is triggered with the ``/session_start`` intent message.
+
+You can define the period of inactivity after which a new conversation
+session is triggered in the domain under the ``session_config`` key.
+``session_expiration_time`` defines the time of inactivity in minutes after which a
+new session will begin. ``carry_over_slots_to_new_session`` determines whether
```

Comment:
Just a thought -- I can already forsee people asking to configure which slots are carried over (e.g. for making reservations, user's name and phone number should be carried over, but the restaurant info for the form shouldn't)

Reply:
this could also be done relatively easily in the override, right? maybe we add that as an example in the note

Reply:
exactly which slots are carried over can be customised in the override. I'll add an example 👍

Reply:
I think it might make more sense to put the override with the custom action docs